cpp/src/aztec/proof_system/flavor/flavor.hpp

```diff
@@ -0,0 +1,169 @@
+              { .name = "W_2",           .num_bytes = g1_size,           .derived_by_verifier = false },
+              { .name = "W_3",           .num_bytes = g1_size,           .derived_by_verifier = false },
+            },
+            /* challenge_name = */ "beta",
+            /* num_challenges_in = */ 2) // also produce "gamma"
+        );
+
+        // Round 3
+        manifest_rounds.emplace_back(transcript::Manifest::RoundManifest(
+            { { .name = "Z_PERM", .num_bytes = g1_size, .derived_by_verifier = false } },
+            /* challenge_name = */ "alpha",
+            /* num_challenges_in = */ 1));
+
+        // Rounds 3 + 1, ... 3 + num_sumcheck_rounds
+        for (size_t i = 0; i < num_sumcheck_rounds; i++) {
+            auto label = std::to_string(num_sumcheck_rounds - i);
+            manifest_rounds.emplace_back(
+                transcript::Manifest::RoundManifest(
+            { 
+              { .name = "univariate_" + label, .num_bytes = fr_size * honk::StandardHonk::MAX_RELATION_LENGTH, .derived_by_verifier = false } 
+            },
+            /* challenge_name = */ "u_" + label,
+            /* num_challenges_in = */ 1));
+        }
+
+        // Rounds 4 + num_sumcheck_rounds
+        manifest_rounds.emplace_back(transcript::Manifest::RoundManifest(
+            {
+              { .name = "w_1",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 0 },
+              { .name = "w_2",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 1 },
+              { .name = "w_3",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 2 },
+              { .name = "sigma_1",      .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 3 },
+              { .name = "sigma_2",      .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 4 },
+              { .name = "sigma_3",      .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 5 },
+              { .name = "q_1",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 6 },
+              { .name = "q_2",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 7 },
+              { .name = "q_3",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 8 },
+              { .name = "q_m",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 9 },
+              { .name = "q_c",          .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 10 },
+              { .name = "z_perm",       .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = 11 },
+              { .name = "z_perm_omega", .num_bytes = fr_size, .derived_by_verifier = false, .challenge_map_index = -1 },
```

Comment:
I dont think `z_perm_omega` (or the appropriate analog for Honk) will exist here right? And maybe this will include id polys and maybe a couple of lagrange polys? Probably not necessary to hammer out for this PR

Reply:
Indeed.